search/quality_eval.go

```diff
@@ -1476,6 +1477,11 @@ func evalResultToHitSources(result EvalResult) ([]HitSource, error) {
 					}
 					hitSource.CarrouselHitSources = append(hitSource.CarrouselHitSources, carrouselHitSource)
 				}
+			} else if hit.Type == consts.SEARCH_RESULT_LESSONS_SERIES_BY_SOURCE || hit.Type == consts.SEARCH_RESULT_LESSONS_SERIES_BY_TAG {
```

Comment:
You generate a new result type here?
What about frontend? How it looks like?

Reply:
https://github.com/Bnei-Baruch/kmedia-mdb/tree/AR-755_group_selected_study_series_in_search_results